Uchu.World/Systems/Behaviors/ChangeOrientationBehavior.cs

```diff
@@ -4,6 +4,40 @@ namespace Uchu.World.Systems.Behaviors
 {
     public class ChangeOrientationBehavior : BehaviorBase
     {
+        private float Angle { get; set; }
+        private float Duration { get; set; }
+        private bool OrientCaster { get; set; }
+        private bool Relative { get; set; }
+        private bool ToAngle { get; set; }
+        private bool ToPoint { get; set; }
+        private bool ToTarget { get; set; }
         public override BehaviorTemplateId Id => BehaviorTemplateId.ChangeOrientation;
+        public override async Task BuildAsync()
+        {
+            OrientCaster = await GetParameter<int>("orient_caster") == 1;
+            ToTarget = await GetParameter<int>("to_target") == 1;
+        }
+
+        public override void ExecuteStart(BehaviorExecutionParameters parameters)
+        {
+            parameters.Context.Associate.Transform.LookAt(parameters.BranchContext.Target.Transform.Position);
+            //the variables seem to do nothing
```

Comment:
If these do nothing, is there a reason these are included, or are we missing something?